submissions/adeyemiJuwon/phpDatabase/orders.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+include('connect.php');
+if( isset($_GET['id']) && strlen($_GET['id']) > 0 ) {
+     $id =  $_GET['id'];
+}
+
+try{
+     $sql = "SELECT order_items.unit_price, 
+            order_items.total_amount, orders.order_date, orders.id ,customers.street_address, customers.city,customers.state, 
+            customers.country,order_items.quantity, products.name
+            FROM order_items 
+            LEFT JOIN orders ON orders.id = order_items.order_id
+            LEFT JOIN customers on orders.customer_id = customers.id
+            LEFT JOIN products ON products.id = order_items.product_id 
+            WHERE  orders.customer_id = '$id'";
+            $results = $pdo->query($sql);
+            setcookie('order_items',  json_encode($results), time() + 3600);
+
+if($results ->rowCount() > 0) {
```

Comment:
You still do not check if cookie is set. The `$result` used here is that gotten each time the query is run.

Reply:
Yes
sorry i forget to change it on my repo
done now
Thank you
@adaugochi

Reply:
I didn't see what you did. Nothing changed. it is still the same

Reply:
Okay
@adaugochi 
Help me to recheck it again, whether it has changed.
Thank you.